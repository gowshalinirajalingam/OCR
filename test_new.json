[
      {
            "Page_Number": 1,
            "Table": [
                  {
                        "SERVICE_CODE": "CONSULTATIONAND SERVICES CONSULTATION CC2206",
                        "QUANTITY": null,
                        "TOTAL": "80.00 1 80.00"
                  },
                  {
                        "SERVICE_CODE": "CONSULTATION -REPEAT VISIT",
                        "QUANTITY": null,
                        "TOTAL": "80.00"
                  },
                  {
                        "SERVICE_CODE": "Subtotal Charges (before Government Subsidy)",
                        "QUANTITY": null,
                        "TOTAL": "-48.00"
                  },
                  {
                        "SERVICE_CODE": "Less: Government Subsidy Subtotal Charges (after Government Subsidy)",
                        "QUANTITY": null,
                        "TOTAL": "32.00 32,00"
                  },
                  {
                        "SERVICE_CODE": "Total Charges Payable",
                        "QUANTITY": null,
                        "TOTAL": "32.00 2.24 34.24 -2.24"
                  },
                  {
                        "SERVICE_CODE": "AMOUNT PAYABLE BEFORE TAX ADD : 7% GST AMOUNT PAYABLE AFTER TAX LESS : GST ABSORBED BY THE GOVERNMENT NET AMOUNT PAYABLE",
                        "QUANTITY": null,
                        "TOTAL": "32.00 32.00 0.00"
                  },
                  {
                        "SERVICE_CODE": "MEDISAVE FOR FLEXI CLAIM PAYMENT MEDISAVE FOR FLEXI CLAIM",
                        "QUANTITY": null,
                        "TOTAL": null
                  },
                  {
                        "SERVICE_CODE": "MSVFI EXI",
                        "QUANTITY": null,
                        "TOTAL": null
                  },
                  {
                        "SERVICE_CODE": "ST: P Holder Name MSV a/c No MSV Deducted Date Time Location 30.11.2018 1220.brs",
                        "QUANTITY": null,
                        "TOTAL": null
                  },
                  {
                        "SERVICE_CODE": "Flexi MSV Deducted",
                        "QUANTITY": null,
                        "TOTAL": null
                  },
                  {
                        "SERVICE_CODE": "* You are served by? Your next appointment.",
                        "QUANTITY": null,
                        "TOTAL": null
                  }
            ],
            "Key_Values": {
                  "Bill_Ref_Number": "Flexi MSV Deducted",
                  "Tax_Invoice_Date": "R",
                  "Visit_Date": ":28.09.2018 0950 hrs",
                  "Payment_Class": ": Cash/Credit",
                  "GST_REG_NO": " : M2-0088821-9",
                  "Page": " 1 of 1",
                  "Total_Charges_Payable": "32.00"
            }
      }
]