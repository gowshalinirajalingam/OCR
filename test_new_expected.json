        [
      {
            "Page_Number": 1,
            "Table": [
                    {
                        "SERVICE_CODE": "CONSULTATIONAND SERVICES",
                        "DESCRIPTION":null,
                        "QUANTITY": null,
                        "TOTAL": "null"
                  },
                  {
                        "SERVICE_CODE": null,
                        "DESCRIPTION":"CONSULTATION",
                        "QUANTITY": "Subtotal",
                        "TOTAL": "80.00"
                  },
                  {
                        "SERVICE_CODE": "CC2206" ,
                        "DESCRIPTION":"CONSULTATION -REPEAT VISIT",
                        "QUANTITY": 1,
                        "TOTAL": "80.00"
                  },
                  {
                        "SERVICE_CODE": null,
                        "DESCRIPTION":null,
                        "QUANTITY": "Subtotal Charges (before Government Subsidy)",
                        "TOTAL": "-48.00"
                  },
                  {
                        "SERVICE_CODE": null,
                        "DESCRIPTION":null,
                        "QUANTITY": "Less: Government Subsidy",
                        "TOTAL": "32.00"
                  },
                  {
                        "SERVICE_CODE": null ,
                        "DESCRIPTION":null,
                        "QUANTITY": "Subtotal Charges (after Government Subsidy)",
                        "TOTAL": "32.00"
                  },
                  {
                        "SERVICE_CODE": null,
                        "DESCRIPTION":null,
                        "QUANTITY": "Total Charges Payable",
                        "TOTAL": "32.00"
                  },
                  {
                        "SERVICE_CODE": "AMOUNT PAYABLE BEFORE TAX ADD : 7% GST AMOUNT PAYABLE AFTER TAX LESS : GST ABSORBED BY THE GOVERNMENT NET AMOUNT PAYABLE",
                        "DESCRIPTION":null,
                        "QUANTITY": null,
                        "TOTAL": "32.00 32.00 0.00"
                  },
                  {
                        "SERVICE_CODE": "AMOUNT PAYABLE BEFORE TAX",
                        "DESCRIPTION":null,
                        "QUANTITY": null,
                        "TOTAL": "32.00"
                  },
                  {
                        "SERVICE_CODE": "ADD : 7% GST",
                        "DESCRIPTION":null,
                        "QUANTITY": null,
                        "TOTAL": "2.24"
                  },
                  {
                        "SERVICE_CODE": "AMOUNT PAYABLE AFTER TAX",
                        "DESCRIPTION":null,
                        "QUANTITY": null,
                        "TOTAL": "34.24"
                  },
                  {
                        "SERVICE_CODE": "LESS : GST ABSORBED BY THE GOVERNMENT",
                        "DESCRIPTION":null,
                        "QUANTITY": null,
                        "TOTAL": "-2.24"
                  },
                  {
                        "SERVICE_CODE": "NET AMOUNT PAYABLE",
                        "DESCRIPTION":null,
                        "QUANTITY": null,
                        "TOTAL": "32.00"
                  },
            
                  {
                        "SERVICE_CODE": "PAYMENT MEDISAVE FOR FLEXI CLAIM",
                        "DESCRIPTION":null,
                        "QUANTITY": null,
                        "TOTAL": 32.00
                  },
                  {
                        "SERVICE_CODE":  null,
                        "DESCRIPTION":null,
                        "QUANTITY": null,
                        "TOTAL": 0.00
                  },
                  {
                        "SERVICE_CODE": "MSVFI EXI",
                        "DESCRIPTION":null,
                        "QUANTITY": null,
                        "TOTAL": null
                  },
                  {
                        "SERVICE_CODE": "ST: P",
                        "DESCRIPTION":null,
                        "QUANTITY": null,
                        "TOTAL": null
                  },
                  {
                        "SERVICE_CODE": "Holder Name",
                        "DESCRIPTION":null,
                        "QUANTITY": null,
                        "TOTAL": null
                  },
                  
                  {
                        "SERVICE_CODE": null ,
                        "DESCRIPTION":"MSV a/c No MSV Deducted Date Time Location 30.11.2018 1220.brs Flexi MSV Deducted",
                        "QUANTITY": null,
                        "TOTAL": null
                  },
                  {
                        "SERVICE_CODE": null,
                        "DESCRIPTION":null,
                        "QUANTITY": "R",
                        "TOTAL": null
                  },
                  {
                        "SERVICE_CODE": null,
                        "DESCRIPTION":null,
                        "QUANTITY": "H",
                        "TOTAL": null
                  },
                  {
                        "SERVICE_CODE": "* You are served by? Your next appointment.",
                        "DESCRIPTION":null,
                        "QUANTITY": null,
                        "TOTAL": null
                  }
            ],
            "Key_Values": {
            	   "Tax Invoice Number":701814043610003,
                  "Bill_Ref_Number": "Flexi MSV Deducted",
                  "Tax_Invoice_Date": "28.09.2018 1051 hrs",
                  "Patient NRIC/HRN":"",
                  "Visit_Date": ":28.09.2018 0950 hrs",
                  "Visit/ Bill Location": "CXSURO / CXSUTO/ URO",
                  "Payment_Class": "SUB - OP (N,3,N)",
                  "Type of Supply": "Cash/ Credit",
                  "Page": " 1 of 1"

            }
      }
]
    
